sdk/trace/simple_span_processor_test.go

```diff
@@ -150,6 +151,36 @@ func TestSimpleSpanProcessorShutdownOnEndConcurrency(t *testing.T) {
 	<-done
 }
 
+func TestSimpleSpanProcessorShutdownOnEndRace(t *testing.T) {
+	exporter := &testExporter{}
+	ssp := sdktrace.NewSimpleSpanProcessor(exporter)
+	tp := basicTracerProvider(t)
+	tp.RegisterSpanProcessor(ssp)
+
+	var wg sync.WaitGroup
+	wg.Add(2)
+	go func() {
+		_, span := tp.Tracer("test").Start(context.Background(), "A")
+		defer span.End()
+		wg.Done()
+	}()
+
+	go func() {
```

Comment:
This code is duplicate as line 162 to 166. Instead of this, my suggestion would be to assign this function to a variable and then call that variable in both places(line 162 and 168).
Something like this.
startTracer := func() {
		_, span := tp.Tracer("test").Start(context.Background(), "A")
		defer span.End()
		wg.Done()
	}